napari/_qt/qt_viewer.py

```diff
@@ -735,22 +736,32 @@ def _screenshot_dialog(self):
         if dial.exec_():
             update_save_history(dial.selectedFiles()[0])
 
-    def _open_files_dialog(self, choose_plugin=False):
-        """Add files from the menubar."""
+    def _open_file_dialog_uni(self, caption: str) -> typing.List[str]:
+        """
+        Open dialog to get list of files from user
+        """
         dlg = QFileDialog()
         hist = get_open_history()
         dlg.setHistory(hist)
 
-        filenames, _ = dlg.getOpenFileNames(
-            parent=self,
-            caption=trans._('Select file(s)...'),
-            directory=hist[0],
-            options=(
-                QFileDialog.DontUseNativeDialog
-                if in_ipython()
-                else QFileDialog.Options()
-            ),
-        )
+        open_kwargs = {
+            "parent": self,
+            "caption": caption,
+        }
+        if "PySide6" in QFileDialog.__module__:
```

Comment:
```suggestion
        if "PySide" in QFileDialog.__module__:
```

This bug also affects pyside2:
https://github.com/napari/napari/issues/5508

Reply:
done

Reply:
works for me in pyside2.